grpc/src/main/proto/contents_service.proto

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.projectnessie.api.grpc";
+option java_outer_classname = "ContentsApiProto";
+
+package org.projectnessie.proto;
+
+service ContentsService {
+  rpc getContents (ContentsRequest) returns (Contents) {};
+  rpc getMultipleContents (MultipleContentsRequest) returns (MultipleContentsResponse) {};
+}
+
+enum Dialect {
+  SPARK = 0;
+  DREMIO = 1;
+  PRESTO = 2;
+  HIVE = 3;
+}
+
+message ContentsRequest {
+  ContentsKey contentsKey = 1;
+  string ref = 2;
+  string hashOnRef = 3;
+}
+
+message MultipleContentsRequest {
+  string ref = 1;
+  string hashOnRef = 2;
+  repeated ContentsKey requestedKeys = 3;
```

Comment:
nit: inconsistent order of fields... or was it intentional?

Reply:
you mean in comparison with `ContentsRequest`? yes that's on purpose. The order of fields is mapped in the same way as they are in the `ContentsResource` class